lib/holidays/definition/context/load.rb

```diff
@@ -14,7 +14,7 @@ def call(region)
           target_region_module = Module.const_get("Holidays").const_get(region.upcase)
 
           @definition_merger.call(
-            target_region_module.defined_regions,
+            region,
```

Comment:
This was the main line that made me pause. I just couldn't believe that this wouldn't result in other behavior changes for regions outside of north america. But the tests pass! 

And thinking it through, what we are 'losing' by not passing in `defined_regions` is that we won't be unnecessarily loading the subregions. We don't lose anything in terms of behavior (except for NA, of course, because of your fix) and we gain in performance. ❤️